spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -58,14 +58,12 @@ This is used by the locally running Data Flow Server to store the state of regis
 ```
 $ wget http://repo.spring.io/milestone/org/springframework/cloud/spring-cloud-dataflow-server-kubernetes/1.0.0.M2/spring-cloud-dataflow-server-kubernetes-1.0.0.M2.jar
 
-$ java -jar spring-cloud-dataflow-server-kubernetes-1.0.0.M2.jar
+$ java -jar spring-cloud-dataflow-server-kubernetes-1.0.0.M2.jar --spring.cloud.deployer.kubernetes.memory=768Mi
 ```
 +
-If you are running Kubernetes using vagrant locally you might need to increase the CPU for the deployed apps using the following startup command:
+NOTE: We haven't tuned the memory use of the OOTB apps yet, so to be on the safe side we are increasing the memory for the pods by providing the following property: `--spring.cloud.deployer.kubernetes.memory=768Mi`
 +
-```
-$ java -jar spring-cloud-dataflow-server-kubernetes-1.0.0.M2.jar --spring.cloud.deployer.kubernetes.cpu=1
-```
+NOTE: If you are running Kubernetes using vagrant locally, then you might need to increase the CPU for the deployed apps using the following property: `--spring.cloud.deployer.kubernetes.cpu=1`
```

Comment:
Is this really `1`? The default value for this property shows `250m` and that made me ask this question.